packages/truffle-solidity-loader/package.json

```diff
@@ -3,18 +3,24 @@
   "version": "0.0.8",
   "description": "A Webpack loader that uses Truffle to load and provision solidity contracts, with Truffle migrations.",
   "main": "index.js",
-  "scripts": {},
-  "author": "John McDowall <john@kantan.io>",
+  "scripts": {
+    "lintjs": "standard",
+    "lintjs-fix": "standard --fix"
+  },
+  "authors": [
+    "John McDowall <john@kantan.io>",
+    "Angello Pozo <angellopozo@gmail.com>"
+  ],
   "license": "MIT",
   "dependencies": {
     "chalk": "^1.1.3",
-    "ether-pudding": "^3.2.0",
     "find-up": "^1.1.2",
-    "findup": "^0.1.5",
-    "lodash.merge": "^4.6.0",
-    "solidity-parser": "^0.1.0",
-    "temp": "^0.8.3",
-    "truffle": "^2.0.8",
-    "web3": "^0.17.0-alpha"
+    "loader-utils": "^1.1.0",
+    "schema-utils": "^1.0.0",
+    "truffle-config": "^1.0.6",
+    "truffle-core": "^4.1.13"
```

Comment:
Hm. I think the benefits outweighs the drawback here. Users using this will likely have Truffle installed anyway. Or if they don't, this is going to force it to be installed. 

By making it a peerDependency, we won't have to update the listed version internally. And since I've been doing a bunch of `yarn.lock` maintenance lately, I'd rather avoid that :)

Reply:
Although I guess I'm a bit conflicted. If it were only `truffle-core`, it'd be a bit less effort on the user's part. We can leave these for now. I might change it later if I notice a bunch of lockfile overhead.

Reply:
Well even if truffle is installed, the project may not run. I think most people have truffle installed globaly, not locally. Which means the peer dependency will not be met. 

For example, I have truffle installed globally and i got an error when i changed to a peer dependency.